### DISCUSSION TITLE
Problem when viewing

### DISCUSSION THREAD

**Discussion by ragnarokme** [jellyfin/jellyfin#9834] [General]
Hello, I have a problem, when trying to watch a movie weighing 95 GB, the video stutters, how can I fix it? Thank you. I am use Debian 11 server

**Comment by El-Biiscuit** [comment]
Hello I have the same problem with my ubuntu 22.04 server and jellyfin is in docker conteners

**Reply by ragnarokme** [reply]
I spat, deleted jellyfin, raised minidlna+samba on my server and installed BubbleUPnP as a proxy, now I'm watching in the Vimu Media player on android TV, I just need to test a movie to find it with a bitrate of 70 mb/sec, if you want I can write about the result, in minidlna I commented (read) turned on the inotify parameter and I changed something else, but there's a little bit, while the flight is normal
